cws-engine-service/src/main/java/jpl/cws/engine/WorkerExternalTaskLockDaemon.java

```diff
@@ -29,8 +29,8 @@
     private Logger log;
 
     private static final long SLEEP_MS       = 15 * 60 * 1000L;	// 15 minutes
```

Comment:
```suggestion
    private static final long SLEEP_MS       = 5 * 60 * 1000L;	// 5 minutes
```

We need to make sure that the lock extender actually extends the lock before it expires. If tasks are only locked for 10 minutes, we need to check more frequently than that. `SLEEP_MS` determines how frequently we check if a task should have its lock extended.